src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetStatisticsRDD.scala

```diff
@@ -301,6 +302,17 @@ private[parquet] object ParquetStatisticsRDD {
     s"filter_${UUID.randomUUID}_block${blockSuffix}_col_${colSuffix}"
   }
 
+  /**
+   * Get FileSystem based on URI
```

Comment:
You should add something like that: Returns file system for FILTER_DIR path, if provided, otherwise, the default file system based on Hadoop configuration.